src/components/picker/multi-range-slider/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { FC, useCallback, useContext, useMemo } from 'react';
 import { Box } from 'rebass';
-import { Range } from 'rc-slider';
+import RcSclider from 'rc-slider';
```

Comment:
single default export component which accecpts `range` prop instead to create RangeSliders